manager/manager.go

```diff
@@ -121,6 +121,9 @@ type Config struct {
 
 	// PluginGetter provides access to docker's plugin inventory.
 	PluginGetter plugingetter.PluginGetter
+
+	// FIPS specifies whether the manager should run in FIPS-compliant mode.
```

Comment:
nit: does it make sense to say that the mode is specified during startup and can't be changed ?